fabulaws-config.yml

```diff
@@ -181,6 +202,12 @@
       db-slave: m4.4xlarge
       web: c3.2xlarge # for faster AMI creation (no swap file)
       worker: m4.xlarge
+    prod01:
```

Comment:
I'd vote for the original prod sizes (half this all around), if not half that. Current prod is quite over-provisioned, and I believe they expect less traffic on this site.